introduction.md

```diff
@@ -0,0 +1,2 @@
+I am Glenn. It is an honor to be part of this dojo.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.